src/Esfa.Vacancy.Azure.Resources/deployments/web-app-resources.json

```diff
@@ -0,0 +1,294 @@
+            "AvmsPlusDatabaseConnectionString": "[variables('registerAppSettings').AvmsPlusDatabaseConnectionString]",
+            "EnvironmentName": "[variables('registerAppSettings').EnvironmentName]",
+            "LoggingRedisKey": "[variables('registerAppSettings').LoggingRedisKey]",
+            "ApprenticeshipInfoApiBaseUrl": "[variables('registerAppSettings').ApprenticeshipInfoApiBaseUrl]",
+            "CacheConnectionString": "[variables('registerAppSettings').CacheConnectionString]",
+            "UseSandboxServices": "[variables('registerAppSettings').UseSandboxServices]"
+          }
+        },
+        {
+          "apiVersion": "2015-08-01",
+          "name": "slotconfignames",
+          "type": "config",
+          "dependsOn": [
+            "[resourceId('Microsoft.Web/Sites', parameters('registerAppName'))]"
+          ],
+          "properties": {
+            "appSettingNames": [ "APPINSIGHTS_INSTRUMENTATIONKEY", "EnvironmentName" ]
+          }
+        },
+        {
+          "apiVersion": "2015-08-01",
+          "name": "[parameters('slotName')]",
+          "type": "slots",
+          "location": "[resourceGroup().location]",
+          "dependsOn": [
+            "[resourceId('Microsoft.Web/Sites', parameters('registerAppName'))]"
+          ],
+          "properties": {},
+          "resources": [
+            {
+              "apiVersion": "2015-08-01",
+              "name": "web",
+              "type": "config",
+              "dependsOn": [
+                "[resourceId('Microsoft.Web/Sites/Slots', parameters('registerAppName'), parameters('slotName'))]"
+              ],
+              "properties": "[variables('siteProperties')]"
+            },
+            {
+              "apiVersion": "2015-08-01",
+              "name": "appsettings",
+              "type": "config",
+              "dependsOn": [
+                "[resourceId('Microsoft.Web/Sites/Slots', parameters('registerAppName'), parameters('slotName'))]"
+              ],
+              "properties": {
+                "APPINSIGHTS_INSTRUMENTATIONKEY": "[reference(resourceId('microsoft.insights/components/', parameters('registerAppName')), '2015-05-01').InstrumentationKey]",
+                "LoggingRedisConnectionString": "[variables('registerAppSettings').LoggingRedisConnectionString]",
+                "AvmsPlusDatabaseConnectionString": "[variables('registerAppSettings').AvmsPlusDatabaseConnectionString]",
+                "EnvironmentName": "[variables('registerAppSettings').EnvironmentName]",
+                "LoggingRedisKey": "[variables('registerAppSettings').LoggingRedisKey]",
+                "ApprenticeshipInfoApiBaseUrl": "[variables('registerAppSettings').ApprenticeshipInfoApiBaseUrl]",
+                "CacheConnectionString": "[variables('registerAppSettings').CacheConnectionString]",
+                "UseSandboxServices": "[variables('registerAppSettings').UseSandboxServices]"
```

Comment:
Use sandbox service needs to be a slot setting, so we can turn it on for Staging and keep it off for prod when time comes.

Reply:
I'll update that to be a slot setting. Cheers

Reply:
Updated